modules/store/spec/runtime_checks.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('Runtime checks:', () => {
     });
   });
 
-  describe('USER_RUNTIME_CHECKS Token', () => {
+  xdescribe('USER_RUNTIME_CHECKS Token', () => {
```

Comment:
Any idea why these tests are failing? 
Could this be an issue with the jest-angular-preset lib?🤨

Reply:
Not sure if its that package. From what I saw it was because of the way `spyOn` is being used, but I was going to look into it further

Reply:
I tried to resolve them by using the jest functions, but I encountered the following next.

```
TestBed : TypeError: Cannot read property 'assertPresent' of undefined
```

There were some issues opened in the Angular repo from a couple years ago with the same problem, e.g. https://github.com/angular/angular/issues/11317